build/resources/main/view/PersonListCard.fxml

```diff
@@ -31,6 +31,7 @@
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="remark" styleClass="cell_small_label" text="\$remark" />
```

Comment:
Do we need PersonListCard.fxml?

Reply:
Will not touch. should be handled by #20